internal/policy/policy.go

```diff
@@ -818,7 +819,122 @@ func ListRules(ctx context.Context, repo *git.Repository, targetRef string) ([]*
 		}
 	}
 
-	return allDelegations, nil
+	rootMetadata, err := state.GetRootMetadata()
```

Comment:
ListRules returning the root metadata is a bit counterintuitive, no? I wonder if the consumer for this can load the root metadata separately, so this func is tighter.